src/Stratis.Bitcoin.Features.Consensus/ConsensusManager.cs

```diff
@@ -18,24 +18,22 @@ public class ConsensusManager : INetworkDifficulty, IGetUnspentTransaction
 
         public Network Network { get; private set; }
 
-        public IPowConsensusValidator ConsensusValidator { get; private set; }
 
-        public ConsensusManager(IConsensusLoop consensusLoop = null, IDateTimeProvider dateTimeProvider = null, NodeSettings nodeSettings = null, Network network = null,
-            IPowConsensusValidator consensusValidator = null)
+        public ConsensusManager(IConsensusLoop consensusLoop = null, IDateTimeProvider dateTimeProvider = null, NodeSettings nodeSettings = null, Network network = null)
```

Comment:
None of them can be null

Reply:
The only thing changed on this constructor was the removal of the consensusValidator - but more than happy to un optional them all if that is the right course of action